docs/localization.md

```diff
@@ -4,37 +4,82 @@ During development, using [`NSLocalizedString()`](https://developer.apple.com/do
 
 During the release process, `NSLocalizedString` statements are scanned and stored in the `Localizable.strings` file. The file is then uploaded to [GlotPress](https://translate.wordpress.org/projects/apps/ios/) for translation. Before the release build is finalized, all the translations are grabbed from GlotPress and saved back to the `Localizable.strings` files.
 
+## Use unique reverse-DNS naming style Keys
+
+Use unique reverse-DNS naming style keys for localized strings (instead of using the English copy as key) to avoid issues where the same word in English could need different translations based on context or long keys being uncomfortable for translators.
```

Comment:
```suggestion
Use unique reverse-DNS naming style for keys o localized strings (instead of using the English copy as key). This allows to avoid issues where the same word in English could need different translations based on context, or very long keys causing issues with some translation services.
```